tests/playground.html

```diff
@@ -142,8 +146,14 @@
             maxScale: 4,
             minScale: 0.25,
             scaleSpeed: 1.1
-          }
+          },
+        plugins: {
+          'toolbox': 'newToolbox',
+          'connectionManager': 'draggedConnections',
```

Comment:
Okay if the goal was to demonstrate the use of `workspace.addPlugin`, but would it have been possible to register workspace search in options, eg: 
```
{
  plugins: {
      'workspacesearch': Blockly.WorkspaceSearch
  }
}
```